waspc/examples/todoApp/todoApp.wasp

```diff
@@ -50,7 +50,8 @@ query getTasks {
 }
 
 query getTask {
-  fn: import { getTask } from "@ext/queries.js"
+  fn: import { getTask } from "@ext/queries.js",
+  entities: [Task]
```

Comment:
just looking at this, might be cool that we have `usingEntities` instead of `entities`? And for actions we might have e.g. `changingEntities` or `affectingEntities`? Not for now of course, but just an idea :D

Reply:
Yes we might do smth like that, could be nice!